packages/esm-patient-list-app/src/PatientTable/patient-table.component.tsx

```diff
@@ -0,0 +1,140 @@
+import { navigate } from '@openmrs/esm-framework';
+import { Link } from 'carbon-components-react';
+import DataTable, {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from 'carbon-components-react/lib/components/DataTable';
+import DataTableSkeleton from 'carbon-components-react/lib/components/DataTableSkeleton';
+import Pagination from 'carbon-components-react/lib/components/Pagination';
+import Search from 'carbon-components-react/lib/components/Search';
+import { debounce } from 'lodash';
+import React, { useMemo, CSSProperties } from 'react';
+import styles from './patient-table.component.scss';
+
+const PatientTable: React.FC<PatientTableProps> = ({ patients, columns, search, pagination, isLoading, autoFocus }) => {
+  const rows: Array<any> = useMemo(
+    () =>
+      patients.map((patient) => {
+        const row = {};
+        columns.forEach((column) => {
+          row['id'] = patient['id'] || patient['uuid'];
+          const value = column.getValue ? column.getValue(patient) : patient[column.key];
```

Comment:
`column.getValue?(patient) || patient[column.key]`

Reply:
@nmalyschkin somehow my typescript compiler will complain. I think it's confusing this with being a ternary operator.

![Screenshot 2021-07-19 at 10 27 49 (2)](https://user-images.githubusercontent.com/26084581/126120226-5073ddec-2a56-4926-a237-a1009c07ab88.png)

Reply:
No. You have a syntax error. I think you meant `getValue?.(patient)`. What you wrote misses the alternative branch specified after an `:` (so TS is right).

Reply:
Resolved!